spec/system/task_spec.rb

```diff
@@ -1,36 +1,38 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project)}
+  let(:task) { FactoryBot.create(:task, project_id: project.id)}
```

Comment:
Factoryにassociationを記載することで`project_id`の記載を省略することが可能です。
```suggestion
  let(:task) { create(:task) }
```

Reply:
修正いたしました。
5e9ad4cd0d0b5f2a3324b4c69a7f083917159576